layerInfo.json

```diff
@@ -186,7 +186,7 @@
 				"deps": ["Protocol-Utils", "Server-Shared-Utils"]
 			},
 			"Server-Tools": {
-				"packages": ["tinylicious"],
+				"packages": ["@fluidframework/tinylicious"],
```

Comment:
Is there a reason you're renaming the package?

Reply:
I agree with Tyler here, I don't think we should rename it; unless somehow it not having a scope was an issue to put it in the server release group?

Reply:
https://github.com/microsoft/FluidFramework/blob/d9ac992de2571f1d4018d778375ad2aa401dd7bd/build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts#L500

--

@tylerbutler

I initially changed the package name from `tinylicious` to `@fluidframework/tinylicious` as the policy handler complained with the message above.


<img width="1035" alt="Screenshot 2023-10-27 at 6 30 24 AM" src="https://github.com/microsoft/FluidFramework/assets/111468570/df93e2ab-2ac7-4aec-9636-ed17c74b5f27">



--

As @alexvy86's suggestion, I will add an exception to retain the current package name 

https://github.com/microsoft/FluidFramework/blob/3460e1c64ce53c5e2dfaefa6c3c99d6cbf7dfbc3/fluidBuild.config.cjs#L129